Blackjack 2023.py

```diff
@@ -352,13 +435,16 @@ def game_start(deck):
 
 # _____ Rigged Deck ______ Used for testing :>
```

Comment:
This sorta test-y, run-y code we usually wrap in a block to stop it from being run if you import this file as a library in a different file. You've probably seen it before:

```py
if __name__ == "__main__":
 # ...
````